api/v1beta3/vcdclustertemplate_conversion.go

```diff
@@ -0,0 +1,5 @@
+package v1beta3
```

Comment:
Aniruddha, please update templates/cluster-class-template.yaml file (or) add a new file for v1beta3